installator.sh

```
@@ -0,0 +1,68 @@
+#!/bin/bash
+[ "$UID" -eq 0 ] || { echo "This script must be run as root."; exit 1;}
+echo "Determine how to install TensorFlow: "
+echo "1 virtualenv"
+echo "2 'native' pip"
+echo "3 Docker"
+read mechanismId
+case $mechanismId in
```

If user mistypes or accidentally presses Enter (which happens a lot), the script will silently fail. Or, even worse: it will partially install packages and silently fail then. I suggest creating a function which re-tries reading until user types in something correctly. Or, at the very least, add an error message.